src/ng1/directives/overflowTooltip/singleLineOverflowTooltip.directive.js

```diff
@@ -43,6 +43,16 @@ export default function singleLineOverflowTooltip(safeTimeout, $window, $resize)
             scope.$on('$destroy', function () {
```

Comment:
Instead of using scope on $destroy bind to jquery's $destroy, this will fire when the element data is still available so `element.tooltip('destroy)` should work as expected. That should be the only change required to fix this issue.